src/wallet/wallet2.h

```diff
@@ -849,6 +849,16 @@ namespace tools
 
     std::unordered_map<std::string, ons_detail> get_ons_cache();
 
+    struct batching_detail
+    {
+      std::string address;
+      uint64_t amount;
+    };
+    std::vector<batching_detail> batching_records_cache;
```

Comment:
Would this not be better as an `unordered_map`?

Reply:
I think part of the reason for this is that the current epee serialization that core_rpc_server is doing requires this sort of split up because it doesn't support serializing a dict properly (yay epee).  But that said, this code ought to be able to put it back into an unordered_map rather than keeping it split up.